google-cloud-storage/acceptance/storage/bucket_encryption_test.rb

```diff
@@ -17,8 +17,14 @@
 describe Google::Cloud::Storage::Bucket, :encryption, :storage do
   let(:bucket_name) { "#{$bucket_names[1]}-encryption" }
   let(:bucket_location) { "us-central1" }
-  let(:kms_key) { "projects/helical-zone-771/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-1" }
-  let(:kms_key_2) { "projects/helical-zone-771/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-2" }
+  let(:kms_key) {
+    ENV["GCLOUD_TEST_STORAGE_KMS_KEY_1"] ||
+      "projects/helical-zone-771/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-1"
```

Comment:
I didn't realize we were also hardcoding the project in these strings. Would it also help to set this to storage.project_id?

```ruby
"projects/#{storage.project_id}/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-1"
```

Reply:
Yeah good idea. Done.